README.md

```diff
@@ -14,7 +14,8 @@ shiny.semantic
 [![total](https://cranlogs.r-pkg.org/badges/grand-total/shiny.semantic)](https://CRAN.R-project.org/package=shiny.semantic)
 <!-- badges: end -->
 
-With this library it's easy to wrap Shiny with [Semantic UI components](https://github.com/Semantic-Org/Semantic-UI). Add a few simple lines of code and some CSS classes to give your UI a fresh, modern and highly interactive look.
+With this library it's easy to wrap Shiny with [Semantic UI components](https://github.com/Semantic-Org/Semantic-UI). Add a few simple lines of code and some CSS classes to give your UI a fresh, modern and highly interactive look. 
+This package is best for single page applications or regular Shiny apps. For using Semantic components with complex multi-tab apps or modyfiyng the `shinydashboard` based applications check our other package [semantic.dashboard](https://github.com/Appsilon/semantic.dashboard).
```

Comment:
> This package is best for single page applications

I'm not sure if this is true, you can create quite complex semantic apps without using dashboard, i.e. using router